tests/integration/test_application_variables.py

```diff
@@ -0,0 +1,37 @@
+from app import settings
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestApplicationVariables(IntegrationTestCase):
+
+    def setUp(self):
+        settings.EQ_ENABLE_FLASK_DEBUG_TOOLBAR = True
+        settings.EQ_UA_ID = 'TestId'
+        IntegrationTestCase.setUp(self)
+
+    def tearDown(self):
+        settings.EQ_ENABLE_FLASK_DEBUG_TOOLBAR = False
+        IntegrationTestCase.tearDown(self)
+
+    def test_flask_toolbar_is_displayed(self):
```

Comment:
Or maybe we should have a unit test at least to test that when it is false we don't configure it?